### PR TITLE
Added tagging to Active Session (Like Session Editing)

### PR DESCRIPTION
**Pull request by abhipi**
- Added Tagging to Active Session.
- Displays number of people tagged, and usernames in a list format (can remove).
- UserSearch Component used

https://github.com/nagimonyc/trk-app/assets/75244191/7715b018-7d1b-49d6-9132-8cf61cc20a57

### CODE REVIEW

**Review by redpepper-nag — APPROVED** [review]

File: trk/src/Screens/TabScreens/Profile/Backend/ClimberProfile.js [hunk 1]
```diff
@@ -219,9 +219,9 @@ const ClimberProfile = ({ navigation }) => {
 
         //console.log('Session Timestamp: ', sessionKey);
         const activeSession = {};
-        activeSession[activeSessionTimestamp] = activeClimbs; //For desc ordering of active session taps
-        console.log('Expired Session is: ', expiredSessions);
-        console.log('Active Session is: ', activeSession);
+        activeSession[activeSessionTimestamp] = (activeClimbs && activeClimbs.length > 0 ? activeClimbs.reverse(): []); //For desc ordering of active session taps
```
Comment (VGmt) [inline trk/src/Screens/TabScreens/Profile/Backend/ClimberProfile.js:222]: 👍

File: trk/src/Components/SessionTapHistory.js [hunk 2]
```diff
@@ -94,7 +129,127 @@ const SessionTapHistory = (props) => {
...
+                }
+            };
+            fetchData();
+        }, [activeQR]); 
+
+        useEffect(() => {
+            const updateTaggedDatabase = async () => {
+                if (data && data.length >0) {
+                    try {
+                        await TapsApi().updateTap(data[data.length-1].tapId, {tagged: tagged});
```
Comment (VGmt) [inline trk/src/Components/SessionTapHistory.js:242]: 🙏

File: trk/src/Components/SessionTapHistory.js [hunk 3]
```diff
@@ -120,14 +275,28 @@ const SessionTapHistory = (props) => {
...
                     </View>
-                    {props.isCurrent && 
-                        (<ListHistory
-                        data={climbs.reverse()}
+                    {(props.isCurrent && climbs && climbs.length > 0) && 
+                        (<View style={{display:'flex', flexDirection: 'column', width: '100%'}}>
+                        
+                        <View style={{width: '100%', justifyContent:'center', display:'flex', alignItems: 'center', flexDirection: 'row', padding: 10}}>
+                            <View style={{display:'flex', width:'20%', paddingHorizontal: 10, justifyContent: 'center', alignItems: 'center'}}>
+                                <Text style={{color: 'black', fontWeight: '400'}}>{tagged.length} Tag{tagged.length!=1? 's': ''}</Text>
```
Comment (VGmt) [inline trk/src/Components/SessionTapHistory.js:283]: ooh nice!

**Review by VGmt — APPROVED** [review]
Well done.


- Great show of discipline and care (-> Tag(s), -> list in descending order, db calls in useEffect). Appreciate it.

**Pull request merged**